ROADMAP.md

```diff
@@ -0,0 +1,157 @@
+OpenMDAO 2020 Development Roadmap
+=============================================
+Author: Justin S. Gray   
+Date: 10/24/2019  
+
+This document represents the OpenMDAO development team's perspective on what we feel are the most important areas to focus on for framework development in 2020. 
+It is intended to communicate our current perspective to the OpenMDAO user community, and to provide a basis for their feedback. 
+It is not intended to be all encompassing or a binding development plan, 
+but should be regarded as an indication of the focus of the core development team. 
+
+There are four main topic areas that will be focused on: 
+- Releasing OpenMDAO V3.0
+- Model & Data Visualization 
+- New Differentiation Tools that Reduce User Effort
+- Establishing A Community Contribution Process
+
+----------------------------------------------
+----------------------------------------------
+----------------------------------------------
+# Releasing OpenMDAO V3.0
+
+In January 2020, we plan to release V3.0 of the framework. 
+This update will **NOT** represent significant change to the codebase or functionality, 
+but will include several small but important practical changes: 
+
+## 1) Dropping support for Python 2.0 and supporting only Python 3.6 and greater
```

Comment:
Proposed: add [PEP 484](https://www.python.org/dev/peps/pep-0484/) typing, and test in in CI. Can play nice with [sphinx](https://github.com/agronholm/sphinx-autodoc-typehints), actually reducing docs burden.